conf/software/LOCAL.mattxps.json

```diff
@@ -0,0 +1,73 @@
+{
+    "linuxbrew_home"            : "/home/linuxbrew/.linuxbrew",
+    "hps_dir"                   : "/home/matthieu/tmp/hps",
+    "warehouse_dir"             : "/home/matthieu/tmp/warehouse/compara/",
```

Comment:
I understand where these come from, but I'm always a little bit against "proprietary" configurations (should we end up with one per Compara member?). So I'm opening a discussion here: should this file be converted to a LOCAL template, maybe renaming it `LOCAL.template.json` and replace `matthieu` by `$USER`, or is it ok to have one LOCAL JSON file per member/laptop?

Reply:
The content itself of `LOCAL.mattxps.json` is actually very similar to the other files. I think both the LSF and SLURM files could be considered as "templates" for your own laptop, but you'd have to remember how to name the file.

Reply:
Then we can leave it as it is :+1: